components/lib/crash/src/main/java/mozilla/components/lib/crash/handler/CrashHandlerService.kt

```diff
@@ -29,10 +31,18 @@ class CrashHandlerService : IntentService(WORKER_THREAD_NAME) {
 
         intent.extras?.let { extras ->
             val crash = Crash.NativeCodeCrash.fromBundle(extras)
+            val skipPrompt = extras.getBoolean(Crash.INTENT_EXTRA_SKIP_PROMPT, false)
+
+            if (skipPrompt) {
+                val notificationManager = getSystemService(Context.NOTIFICATION_SERVICE)
+                        as NotificationManager
+                notificationManager.cancelAll()
```

Comment:
Cancelling all notification is a bit aggressive here.

Reply:
I'll look into if I can just cancel the crash notification.  Thanks,